src/MoBi.Core/Commands/BuildingBlockChangeCommandBase.cs

```diff
@@ -21,9 +24,11 @@ protected BuildingBlockChangeCommandBase(T buildingBlock)
 
       protected override void ExecuteWith(IMoBiContext context)
       {
+         var originalPkSimModuleState = _buildingBlock.Module?.IsPKSimModule ?? false;
```

Comment:
`buildingBlock.Module?.IsPKSimModule ?? false` is used 2 times here and in `shouldConvertToExtensionModule()` function below.
Would encapsulate it in a property of `IBuildingBlock`

Reply:
If it goes an extension of Module then we can probably hide all the null operators in the extension method.

Reply:
or like this, yes

Reply:
If we do it as an extension of the MODULE instead of BUILDINGBLOCK, it maybe confusing. 
Since the Module has the property IsPkSimModule, therefore, adding a method which would return the same value (similar name and information) may be confusing, would you agree? 
I added it at a building blokc level, but of course, feel free to point out whatever you think about it.

Reply:
Agree.